### PR TITLE
Fix broken markdown links

### PR DESCRIPTION
**Pull request by cydu-cloud**
Fixed all the existing broken links.

Signed-off-by: Chuanying Du <cydu@google.com>

### CODE REVIEW

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by mazzystr** [comment]
@cydu-cloud Thanks so much for this contribution.  This is going to help a lot!

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Check docs links
```
[the remark this answers is not in the document]
Comment (cydu-cloud) [reply .github/workflows/action.yml]: SG, removed this file. 

Get this fix in first to fix existing broken links, and then work on the prow integration.

**Comment by mazzystr** [comment]
/lgtm
/approve

@cydu-cloud Thanks!  I hope you stick around for awhile :)

**Comment by mazzystr** [comment]
@cydu-cloud Hello, I'm looking for a contact at Google that can help me with authorization in getting Google added to the KubeVirt ADOPTERS file and display a logo on https://kubevirt.io.  Please contact me via the kubevirt-dev mailing list.  Thanks!

**Pull request merged**